spec/abci++/abci++_methods_002_draft.md

```diff
@@ -289,7 +289,7 @@ title: Methods
     |-------------------------|---------------------------------------------|------------------------------------------------------------------------------------------------------------------|--------------|
     | max_tx_bytes            | int64                                       | Currently configured maximum size in bytes taken by the modified transactions.                                   | 1            |
     | txs                     | repeated bytes                              | Preliminary list of transactions that have been picked as part of the block to propose.                          | 2            |
-    | local_last_commit       | [ExtendedCommitInfo](#extendedcommitinfo)   | Info about the last commit, obtained locally from Tendermint's data structures.                                  | 3            |
```

Comment:
Why this change?

Reply:
Could be, the last three ring a bell TBH. But since I have several PRs advancing in parallel, I lost track what's in and what not. Sorry if this caused you trouble